controller/k8s/injector_test.go

```diff
@@ -0,0 +1,60 @@
+package k8s
+
+import (
+	"bufio"
+	"bytes"
+	"os"
+	"strings"
+	"testing"
+)
+
+var injectEnv = `
+       - env:
+         - name: CONDUIT_PROXY_LOG
+           value: warn,conduit_proxy=info
+         - name: CONDUIT_PROXY_CONTROL_URL
+           value: tcp://proxy-api.conduit.svc.cluster.local:5555
+         - name: CONDUIT_PROXY_CONTROL_LISTENER
+           value: tcp://0.0.0.0:4190
+         - name: CONDUIT_PROXY_PRIVATE_LISTENER
+           value: tcp://127.0.0.1:4143
+         - name: CONDUIT_PROXY_PUBLIC_LISTENER
+           value: tcp://0.0.0.0:4140
+         - name: CONDUIT_PROXY_NODE_NAME
```

Comment:
consider testing the entire yaml file. similar to `install_test.go`: https://github.com/runconduit/conduit/blob/277c06cf1e1e3116765857aed6da3a2a05d7503a/cli/cmd/install_test.go#L13